test/integration/tests/tribalchief.ts

```diff
@@ -0,0 +1,100 @@
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { solidity } from 'ethereum-waffle';
+import { ethers } from 'hardhat';
+import { NamedContracts } from '@custom-types/types';
+import proposals from '@test/integration/proposals_config';
+import { TestEndtoEndCoordinator } from '@test/integration/setup';
+import mainnetAddresses from '@protocol/mainnetAddresses';
+import tribalchief from '@protocol/tribalchief';
+
+describe('e2e-tribalchief', function () {
+  let contracts: NamedContracts;
+  let e2eCoord: TestEndtoEndCoordinator;
+  const doLogging = Boolean(process.env.LOGGING);
+
+  before(async () => {
```

Comment:
I've seen this pattern elsewhere, not specific to this PR, but why do we set `chai.use()` in a `before()` block? Why not just at the top level beneath imports? It's test infrastructure setup, rather than unit test logic

Reply:
it may have something to do with tests running in //, idk